Jointify/Views/Screens/ProcessingView.swift

```diff
@@ -116,17 +118,31 @@ struct ProcessingView: View {
                 print("Analysing frame \(frameCount)/\(frames.count)")
                 
                 let drawnImage = poseNet.predict(frame)
+                
+                let outputQualityAcceptable = poseNet.assessOutputQuality()
 
-                returnMeasurementFrames.append(
-                    MeasurementFrame(
-                        degree: poseNet.calcAngleBetweenJoints(),
-                        image: drawnImage
+                // Only append measurement frame if it fulfills quality criteria
+                if outputQualityAcceptable {
+                    self.removedFramesCounter += 1
```

Comment:
If the output quality is acceptable the removedFramesCounter goes up? 🤔

Reply:
oh wow, wrong variable name but correct logic :D fixed!